perf/benchmark/README.md

```diff
@@ -172,34 +170,39 @@ fortioclient   LoadBalancer   xxxx          xxxx       8080:31759/TCP,8079:30495
 export FORTIO_CLIENT_URL=http://$(kubectl get services -n twopods fortioclient -o jsonpath="{.status.loadBalancer.ingress[0].ip}"):8080
 ```
 
-2. Set `PROMETHEUS_URL`: 
+or if you don't have an external IP:
 
 ```bash
-kubectl -n istio-prometheus port-forward $(kubectl -n istio-prometheus get pod -l app=prometheus -o jsonpath='{.items[0].metadata.name}') 9090:9090 & 
+kubectl -n twopods port-forward svc/fortioclient 8080:8080 &
+export FORTIO_CLIENT_URL=http://localhost:8080
+```
+
+2. Set `PROMETHEUS_URL`:
 
-export PROMETHEUS_URL=http://localhost:9090 
+```bash
+kubectl -n istio-prometheus port-forward svc/prometheus 9090:9090 &
```

Comment:
@johscheuer this command is not working for me, the one you deleted works. Here is the error I got:
```
(runner) bash-3.2$ kubectl -n istio-prometheus port-forward svc/prometheus 9090:9090 &
[1] 52084
(runner) bash-3.2$ Error from server (NotFound): services "prometheus" not found
```

Reply:
Could you show the result of `kubectl -n istio-prometheus get svc` ?

Reply:
I just looked at the default chart which names the Prometheus service like this : https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/prometheus/templates/service.yaml#L4

Reply:
Ah I see it is named `istio-prometheus` here: https://github.com/istio/tools/blob/master/perf/prometheus-install/prometheus.yaml#L129

Reply:
```
$ kubectl -n istio-prometheus get svc
NAME                  TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
istio-prometheus      ClusterIP   10.99.3.36   <none>        9090/TCP   120m
prometheus-operated   ClusterIP   None         <none>        9090/TCP   120m
prometheus-operator   ClusterIP   None         <none>        8080/TCP   120m
```

Reply:
I created a PR to correct the command: https://github.com/istio/tools/pull/331 sorry for the circumstances but there are so many different ways how people install stuff. I adjusted the readme to match the install method otherwise a user should know how Prometheus was installed.